internal/controller/storage/storage.go

```diff
@@ -0,0 +1,95 @@
+package storage
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
+
+	danaiov1alpha1 "github.com/dana-team/nfspvc-operator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	nfsPvcDanaLabel = "nfspvc.dana.io/nfspvc-owner"
+)
+
+func PreparePVC(nfspvc danaiov1alpha1.NfsPvc, StorageClass string) corev1.PersistentVolumeClaim {
```

Comment:
Missing comment

Reply:
It doesn't actually create a new PVC, it just returns a PersistentVolumeClaim object as per given parameters